CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Changes
 
 #### Improvements
+
+- index added to the timestamp column
```

Comment:
This introduced a new schema for the database. I think it should be specified.

Reply:
Can you clarify what you mean when you ask for the new schema to be specified?

To my understanding, that is what I am saying here: I have added an index to the timestamp column.

Reply:
Maybe a more explicit sentence could be great, something like: "feat(models): index added to `LogEntry.timestamp`".
I like to see which part of the library is modified when I read changelogs.

Reply:
gotcha, how about now?